buildingspy/development/check_parameters.py

```diff
@@ -0,0 +1,476 @@
+#!/usr/bin/env python
+###########################################################
+# Script with functions to validate consistency
+# between .mos files and .mo files.
+
+# This script checks invalid expression (x*y) in 
+# the experiment annotation of a Modelica model,
+# In addition it checks if an experiment annotation exists
+# in all Modelica models. 
+# It checks if the parameters of experiment annotation is 
+# consistent with the values defined in the mos script.
+# It checks if each model has a tolerance less than 1e-6.
+# It checks it the mos script contains expression such as
+# startTime=startTime.
+# If any of the checks fail, the script returns with an error.
+#
+# TSNouidui@lbl.gov                            2017-01-24
+###########################################################
+#
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import logging as log
+
+log.basicConfig(filename='check_parameters.log', filemode='w',
+                level=log.DEBUG, format='%(asctime)s %(message)s',
+                datefmt='%m/%d/%Y %I:%M:%S %p')
+stderrLogger = log.StreamHandler()
+stderrLogger.setFormatter(log.Formatter(log.BASIC_FORMAT))
+log.getLogger().addHandler(stderrLogger)
+
+import os
+import re
+import sys
+
+class Validator(object):
```

Comment:
Shouldn't this code be in `buildingspy/development/validator.py`, as this file already has a `Validator` class.